common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -479,7 +480,9 @@ def test_course_structure_view(self):
         And I see correct content in the notes and groups
         """
         self._add_default_notes()
-        self.notes_page.visit().switch_to_tab("structure")
+        self.notes_page.visit()
+        self.notes_page.wait_for_page()
```

Comment:
The visit method [includes a wait_for_page](https://github.com/edx/bok-choy/blob/master/bok_choy/page_object.py#L264), so you should not need to call it explicitly. If it's required, then there's something else going wrong...